events/yaml-sync.js

```diff
@@ -5,31 +5,25 @@ const fs = require('fs')
 const path = require('path')
 
 const p = path.join(__dirname, '..', 'locale', 'en', 'get-involved', 'events.md')
-const lines = fs.readFileSync(p).toString().split('\n')
-const begin = lines.indexOf('---') + 1
-const end = lines.indexOf('---', begin)
-const store = yaml.safeLoad(lines.slice(begin, end).join('\n'))
-
-function getRegion (region) {
-  let reg
-  for (reg in store.regions) {
-    if (store.regions[reg].region === region) return store.regions[reg]
+const contents = fs.readFileSync(p, { encoding: 'utf8' }).trim().slice(4, -4)
```

Comment:
Can you please add a comment about slicing? (YAML head, I know, but it's not that obvious.)